test.bash

```diff
@@ -19,8 +15,8 @@ sleep 20 && solana airdrop 10000
 echo "Starting LiteRpc"
 cargo run &
 
-echo "Running cargo tests in 20s"
-sleep 20 && cargo test
```

Comment:
I think there are other tests that will not be done if you remove cargo tests

Reply:
hm don't think so. cargo test got executed twice in the whole process.
anyway - will revert this - it's not relevant for the PR